README.md

```diff
@@ -22,13 +22,13 @@ Based on and extension of [ngx-translate](https://github.com/ngx-translate/core)
 | 6 - 7           | 1.0.2            | 1.0.1       | legacy |
 | 7               | 1.7.3            | 1.1.0       | legacy |
 | 8               | 2.2.3            | 1.1.0       | legacy |
-| 8 - 12          | 3.1.9            | 1.1.2       | active |
-| 13              | 4.0.1            | 2.0.0       | active |
-| 14              | 5.1.1            | 2.0.0       | active | need rxjs 7 or higher   |
-| 15              | 6.0.0            | 2.0.0       | active | minimum angular 15.0.3  |
-| 15.1            | 6.1.0            | 2.0.0       | active | minimum angular 15.1.0  |
-| 16              | 7.0.0            | 2.0.0       | active | minimum angular 16      |
-| 17              | 7.1.0            | 2.0.0       | active | optional standalone API |
+| 8 - 12          | 3.1.9            | 1.1.2       | legacy |
+| 13              | 4.0.1            | 2.0.0       | legacy |
+| 14              | 5.1.1            | 2.0.0       | legacy | need rxjs 7 or higher   |
+| 15              | 6.0.0            | 2.0.0       | legacy | minimum angular 15.0.3  |
+| 15.1            | 6.1.0            | 2.0.0       | legacy | minimum angular 15.1.0  |
+| 16              | 7.0.0            | 2.0.0       | legacy | minimum angular 16      |
+| 17              | 7.1.0            | 2.0.0       | legacy | optional standalone API |
 | 18              | 7.2.1            | 2.0.0       | active |  |
 
```

Comment:
Maybe add smth like this?

`| 19 | 8.0.0 | 2.0.0 | active | ngx-translate v16 |`